src/Mvc/Mvc.Core/src/PublicAPI.Shipped.txt

```diff
@@ -23,7 +23,7 @@ Microsoft.AspNetCore.Mvc.ActionResult
 Microsoft.AspNetCore.Mvc.ActionResult.ActionResult() -> void
 Microsoft.AspNetCore.Mvc.ActionResult<TValue>
 Microsoft.AspNetCore.Mvc.ActionResult<TValue>.ActionResult(TValue value) -> void
-Microsoft.AspNetCore.Mvc.ActionResult<TValue>.Value.get -> TValue
+Microsoft.AspNetCore.Mvc.ActionResult<TValue>.Value.get -> TValue?
```

Comment:
This file should not be modified

Reply:
💯% agreed.

@pranavkm if the tools aren't doing what we want, manually copy the old lines into the PublicAPI.Unshipped.txt file with the `*REMOVED*` prefix, copy the new lines to the same file (without a prefix😀), then restore this file and CodeCheck.ps1.